src/WebJobs.Script.WebHost/Middleware/HostWarmupMiddleware.cs

```diff
@@ -53,7 +53,7 @@ public class HostWarmupMiddleware
 
         public Task Invoke(HttpContext httpContext)
         {
-            if (IsWarmUpRequest(httpContext.Request, _webHostEnvironment.InStandbyMode, _environment))
+            if (_webHostEnvironment.InStandbyMode)
```

Comment:
Removed this full "IsWarmupRequest" check as it is redundant. We execute this middleware only if that condition matches - code [here](https://github.com/Azure/azure-functions-host/blob/a4ed4428e75aa6547afb1f6b76d219268e12d613/src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs#L71)